src/index.css

```diff
@@ -0,0 +1,71 @@
+:root {
```

Comment:
- [ ] In the mobile view, this page is not rendering well because the content overflows the screen:

![image](https://github.com/CaptainBawa/DocBook-Front-End/assets/57421823/e190491c-4be9-43b5-8b7b-9e6e414b8044)

Kindly, add a little more styling so the info shows correctly. 👍